management/ticket.go

```diff
@@ -13,6 +13,13 @@ type Ticket struct {
 	// applied.
 	TTLSec *int `json:"ttl_sec,omitempty"`
 
+	// ID of the client. If provided for tenants using New Universal Login experience,
+	// the user will be prompted to redirect to the default login route of the
+	// corresponding application once the ticket is used.
+	//
+	// Conflicts with: ResultURL
+	ClientID string `json:"client_id,omitempty"`
```

Comment:
Shouldn't this be `*string` to reflect that it is optional? Right now it will break the API if I'm not mistaken.

Reply:
yes, thank you